src/components/UsernameForm/index.js

```diff
@@ -63,21 +70,41 @@ class UsernameForm extends Component {
     return `/user/${username}`;
   };
 
-  render = () => (
-    <div className="pb-8 md:pt-16">
-      <TimeMessage />
-      <form
-        action="/"
-        className="flex h-8 mx-auto w-5/6 md:w-3/5 lg:w-1/3"
-        method="get"
-        onSubmit={this.handleSubmit}
-        style={formStyle}
-      >
-        <UsernameInput value={this.state.username} onChange={this.handleUsernameChange} />
-        <CheckButton />
-      </form>
-    </div>
-  );
+  /**
+   * Check the condition for eligibility.
+   *
+   * @param {*} pullRequestCount
+   * @param {*} otherReposCount
+   * @returns {boolean}
+   */
+  conditionChecker(pullRequestCount, otherReposCount) {
```

Comment:
Functions are actions so they're usually named prefixed with a "verb" to describe what they do. Better name for this one could be - `checkEligibility`.

Reply:
Thank u dai.